### ISSUE TITLE
Cannot Find Desktop Shortcut for PlayOnLinux Script

### ISSUE THREAD

**Issue by johnstonjs** [mdeguzis/SteamOS-Tools#178]
#### Your system information

* Steam client version: 1511396778
* SteamOS version:  1:2.121
* Opted into Steam client beta?: No
* Opted into SteamOS beta?: No
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible:

Issue: After following the steps [shown here](https://github.com/mdeguzis/SteamOS-Tools/wiki/Playing-Steam-(Windows)-Games-Using-PlayOnLinux) to install Skyrim via PlayOnLinux, the Desktop Shortcut does not show in the _Add Library Shortcut_ screen of SteamOS (Big Picture Mode).

Steam for Windows was installed as user _steam_ and Skyrim works from Desktop Mode.

I suspect there's just an issue with one of the scripts I edited (reproduced below), but I've gone through them dozens of times.  If anyone can help spot the issue with my configuration, please let me know.  I'm happy to provide more details if needed.

The output of `steamos-systeminfo-tool` is attached:
[steamos-systeminfo-stdout.txt](https://github.com/mdeguzis/SteamOS-Tools/files/1503153/steamos-systeminfo-stdout.txt)

Thank you for creating and maintaining such an excellent guide and toolset!

#### Steps for reproducing this issue:

1. Follow [steps to this point](https://github.com/mdeguzis/SteamOS-Tools/wiki/Playing-Steam-(Windows)-Games-Using-PlayOnLinux#install-your-games) to install Skyrim in Steam for Windows via PlayOnLinux.

_Note_: Permissions (and location) for Skyrim executable:
```
-rwxr-xr-x 1 steam steam 1929232 Nov 24 14:35 /home/steam/.PlayOnLinux/wineprefix/Steam/drive_c/Program Files/Steam/steamapps/common/Skyrim/SkyrimLauncher.exe
```

2. Follow [this step](https://github.com/mdeguzis/SteamOS-Tools/wiki/Playing-Steam-(Windows)-Games-Using-PlayOnLinux#adding-your-steam-wine-games-to-steamos--bpm) to generate this execution script (only edited portion shown):
```
# Define some Wine variables for use in the script.
export WINEPREFIX="/home/steam/.PlayOnLinux/wineprefix/Steam"
export WINEDEBUG="-all"
WINEVERSION="/home/steam/.PlayOnLinux/wine/linux-x86/2.12-staging"

# Change LD_LIBRARY_PATH to include the lib directories for the version of wine that you are running.
# Please see: issues/115
export LD_LIBRARY_PATH="$WINEVERSION/lib:$WINEVERSION/lib64:$LD_LIBRARY_PATH"
export PATH="$WINEVERSION/bin:$PATH"

# Define Steam location and game location here.
# NOTE: GAME_DIR is not used with current method. Possibly remove?
STEAM_DIR="$WINEPREFIX/drive_c/Program Files/Steam"
GAME_APPID="72850"
GAME_DIR="steamapps/common/Skyrim"
GAME_EXE="SkyrimLauncher.exe"
```
_Note:_ Permissions for this file:
```
$ ls -l /usr/bin/steam_skyrim 
-rwxr-xr-x 1 root root 3953 Nov 24 19:54 /usr/bin/steam_skyrim
```
3. Follow [this step](https://github.com/mdeguzis/SteamOS-Tools/wiki/Playing-Steam-(Windows)-Games-Using-PlayOnLinux#adding-the-desktop-shortcut) to generate the `.desktop` file.
```
[Desktop Entry]
Version=1.0
Name=Skyrim
GenericName=skyrim
Type=Application
Exec=/usr/bin/steam_skyrim
Icon=playonlinux
Categories=Game;
```

_Note:_ Permissions for this file:
```
$ ls -l /usr/share/applications/skyrim.desktop 
-rw-r--r-- 1 root root 137 Nov 24 19:56 /usr/share/applications/skyrim.desktop
```

4. Return to SteamOS Big Picture Mode and attempt to add non-present Desktop Shortcut

#### Progress Tracker

- [ ] Investigate issue
- [ ] Fix committed
- [ ] Confirmed Resolution

**Comment by johnstonjs** [comment]
And... I figured it out.

It looks like SteamOS will not allow you to add a Desktop Shortcut that has _steam_ anywhere in the Exec line of the `.desktop` file.  This was previously reported [here](https://github.com/ValveSoftware/SteamOS/issues/424), but I didn't notice it previously.

After renaming my execution script:
````
$ ls -l /usr/bin/skyrim 
-rwxr-xr-x 1 root root 3953 Nov 24 19:54 /usr/bin/skyrim
````

and editing the .desktop file
````
[Desktop Entry]
Version=1.0
Name=Skyrim
GenericName=skyrim
Type=Application
Exec=/usr/bin/skyrim
Icon=playonlinux
Categories=Game;
````
I'm now able to add the Desktop Shortcut, though I'm still working on getting Skyrim to actually launch.  Thanks for reading as I figured this out.
